### PR TITLE
sipliy split init.rs into several files.

### PR DESCRIPTION
**Pull request by utam0k**
relate https://github.com/containers/youki/issues/295

### CODE REVIEW

**Comment by utam0k** [comment]
@yihuaf @Furisto @YJDoc2 What do you think of this PR?

**Comment by yihuaf** [comment]
Changes looks good. I think splitting in this case can go either way. On one hand, the file is not that long and having everything in one file actually helps the understanding. I will admit, `init.rs` is probably the wrong name if we keep these functions in one file. On the other hand, I like how you split up the code.

File: src/process/intermediate.rs [hunk 1]
```diff
@@ -0,0 +1,101 @@
+use anyhow::{Context, Result};
+use nix::unistd::{Gid, Uid};
+use oci_spec::runtime::{LinuxNamespaceType};
+
+
+
```
Comment (yihuaf) [inline src/process/intermediate.rs:6]: Nit: that's a lot of empty lines :)

**Review by yihuaf — APPROVED** [review]
As mentioned, the change itself LGTM. I am OK with splitting or not.

**Review by Furisto — APPROVED** [review]

**Review by YJDoc2 — APPROVED** [review]
Looks good :+1:

**Pull request merged**